docs/1.getting-started/7.state-management.md

```diff
@@ -1,10 +1,24 @@
 ---
 navigation.icon: uil:database
-description: Nuxt provides useState composable to create a reactive and SSR-friendly shared state.
+description: Nuxt powerful state management libraries and useState composable to create a reactive and SSR-friendly shared state.
```

Comment:
The sentence now lacks a verb. Probably `provides` should stay insteand of being replaced? Or did you mean something else?

Reply:
you are right, I should have re-read my self